db/migrate/20241021140054_create_api_keys.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class CreateApiKeys < ActiveRecord::Migration[7.1]
+  def up
+    create_table :api_keys, id: :uuid do |t|
+      t.references :organization, type: :uuid, null: false, foreign_key: true
+
+      t.string :value, null: false
```

Comment:
would you also add a unique index scoped to the organization_id?

Reply:
:+1: 

An index on `organization_id` is also mandatory since we are doing `organization.api_keys.first` in multiple places

Reply:
Adding unique index completely make sense. It'd be global rather then scoped by `organization_id` since we need by ApiKey identify only 1 organization.

However, I missed something with @vincent-pochet comment. Index on `organization_id` already in place as part of `t.references`. Maybe you mean some other combination for index?

Reply:
Ah indeed, forget my remark, the index is here. Sorry for the noise ;)